tests/integration/targets/ec2/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+vpc_cidr: '10.{{ 256 | random(seed=resource_prefix) }}.0.0/16'
+subnet_cidr: '10.{{ 256 | random(seed=resource_prefix) }}.1.0/24'
+ec2_ami_name: 'ami-8c1be5f6'
```

Comment:
The risk with hard coding an AMI ID here is that this breaks if we start splitting the tests over multiple regions.  Take a look at the ec2_vol integration tests for an example for how to lookup the AMI by name rather than ID.